docs/docs/rendering-elements.md

```diff
@@ -5,4 +5,76 @@ permalink: docs/rendering-elements.html
 prev: introducing-jsx.html
 ---
 
-TODO
+Elements are the smallest building block of React apps.
+
+An element describes what you want to see on the screen:
+
+```js
+const element = <h1>Hello, world</h1>;
+```
+
+Unlike browser DOM elements, React elements are plain objects, and are cheap to create. React DOM takes care of updating the DOM to match the React elements.
+
+## Rendering an Element into the DOM
+
+Let's say there is a `<div>` somewhere in your HTML file:
+
+```html
+<div id="root"></div>
+```
+
+We call this a "container" DOM node because it serves as an "entry point" for a React app. Everything inside it will be managed by React DOM.
+
```

Comment:
How about:
We call this a "container" DOM node because the React app is contained by it. Everything inside it will be managed by React DOM.